README.md

```diff
@@ -51,26 +52,84 @@ For instructors
 `swc-installation-test-1.py` is pretty simple, and just checks that
 the students have a recent enough version of Python installed that
 they'll be able to parse `swc-installation-test-2.py`.  The latter
-checks for a list of dependencies and prints error messages if a
+checks for your workshop's requirements and prints error messages if a
 package is not installed, or if the installed version is not current
-enough.  By default, the script checks for pretty much anything that
-has ever been used at a Software Carpentry workshop, which is probably
-not what you want for your particular workshop.
-
-Before your workshop, you should go through
-`swc-installation-test-2.py` and comment any dependencies you don't
-need out of the `CHECKS` list.  You might also want to skim through
-the minimum version numbers listed where particular dependencies are
-defined (e.g. `('git', 'Git', (1, 7, 0), None)`).  For the most part,
-fairly conservative values have been selected, so students with modern
-machines should be fine.  If your workshop has stricter version
-requirements, feel free to bump them accordingly.
-
-Similarly, the virtual dependencies can be satisfied by any of several
-packages.  If you don't want to support a particular package (e.g. if
-you have no Emacs experience and don't want to be responsible for
-students who show up with Emacs as their only editor), you can comment
-out that particular `or_dependency`.
-
-Finally, don't forget to post your modified scripts somewhere where
-your students can download them!
+enough.  When you setup your workshop, you should write a [JSON][]
+file listing the lessons you will cover.  For example:
+
+```json
+{
+  "instructions": "https://swcarpentry.github.io/2015-11-09-abc/#setup",
+  "lessons": {
+    "shell": {
+      "requirements": "https://raw.githubusercontent.com/swcarpentry/shell-novice/v5.4/requirements.json"
+    },
+    "git": {
+      "requirements": "https://raw.githubusercontent.com/swcarpentry/git-novice/v5.3/requirements.json"
+    },
+    "python": {
+      "requirements": "https://raw.githubusercontent.com/swcarpentry/python-novice-inflammation/v5.4/requirements.json"
+    },
+    "sql": {
+      "requirements": "https://raw.githubusercontent.com/swcarpentry/sql-novice-survey/v5.7/requirements.json"
+    }
+  }
+}
+```
+
+The lesson names (“shell”, “git”, …) are only used for logging, and
+you may add other attributes or entries that don't define
```

Comment:
Not really clear what "you may add other attributes or entries that don't define `requirements` if you wish" means.